src/ring_theory/tensor_product.lean

```diff
@@ -714,6 +714,46 @@ end
 
 end monoidal
 
-end tensor_product
+section
+
+variables {R A B S: Type*} [comm_semiring R] [semiring A] [semiring B] [comm_semiring S]
+variables [algebra R A] [algebra R B] [algebra R S]
+variables (f : A →ₐ[R] S) (g : B →ₐ[R] S)
+
+variables (S)
+
+/-- The multiplication map on an algebra is a morphism of algebras. -/
+def lmul' : S ⊗[R] S →ₐ[R] S :=
+alg_hom_of_linear_map_tensor_product (algebra.lmul' R)
+  (λ a₁ a₂ b₁ b₂, by simp only [algebra.lmul'_apply, mul_mul_mul_comm])
+  (λ r, by simp only [algebra.lmul'_apply, _root_.mul_one])
+
+variables {S}
+
+lemma lmul'_to_linear_map : (lmul' S).to_linear_map = algebra.lmul' R := rfl
+
+@[simp] lemma lmul'_apply_tmul (a b : S) : lmul' S (a ⊗ₜ[R] b) = a * b := lmul'_apply
+
+include f g
```

Comment:
This is now useless

Reply:
Without this I have to manually unfold `algebra.tensor_product.lmul'` at some places.

Reply:
I mean the `include`